lib/src/main/java/ECSE428Project/controller/AccountController.java

```diff
@@ -99,6 +99,51 @@ public AccountDto updateAccountWithNewEmail(@RequestParam("oldEmail") String old
         return convertToDto(account);
     }
 
+    /**
+     * This method updates the score associated to an account
+     * @param email the email associated to the account
+     * @param scoreString the score that is going to be assigned to the account as a String
+     * @return updated account
+     * @throws ResponseStatusException if the score is not a number
+     */
+    @PutMapping(path = "/account/email/score")
+    public AccountDto assignScoreToAccount(@RequestParam("email") String email, @RequestParam("score") String scoreString)
+            throws ResponseStatusException {
+
+        try {
+            Double.parseDouble(scoreString);

```

Comment:
Why are we parsing it as a double?

Reply:
Maybe a player could get half points or some number of points between 0 and 1 depending on how much time he or she took to answer.
Also it's easier to convert a double to an int than to convert an int to a double so might as well

Reply:
ok for the score it makes sense